library/Requests/Capability.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Capability interface declaring the known capabilities
+ *
+ * @package Requests
+ * @subpackage Capability
+ */
+
+/**
+ * Capability interface declaring the known capabilities
+ *
+ * This is used as discovery to figure out which capabilities can be queried.
+ *
+ * @package Requests
+ * @subpackage Capability
+ */
+class Requests_Capability {
```

Comment:
> Should this be a class or an interface ?

Ideally, it'd be an enum. However, given there's no clean way to do that in PHP 5.6 (at least without additional dependencies), I went with an interface to just keep track of the constants for now. I'll adapt the definition.

> Should the interface extend the Transport interface ?

No, it is not of type "transport", it is just a collection of constants (for now).

> Should the test method stub be moved from the Transport interface to the Capability interface ?

The `test()` method actually does multiple things and is not exclusively tied to testing capabilities, therefore I'd keep it as-is for now. If we ever want to make this cleaner, it would require a more involved refactor.